src/Mock.ts

```diff
@@ -26,23 +28,23 @@ export const JestMockExtended = {
     },
 };
 
-export interface CalledWithMock<T, Y extends any[]> extends jest.Mock<T, Y> {
-    calledWith: (...args: Y | MatchersOrLiterals<Y>) => jest.Mock<T, Y>;
+export interface CalledWithMock<T extends FunctionLike> extends jest.Mock<T> {
+    calledWith: (...args: [...MatchersOrLiterals<Parameters<T>>]) => jest.Mock<T>;
```

Comment:
Changed this to just `MatchersOrLiterals` since that type will check literals for us - the original type for args was `Y | MatchersOrLiterals<Y>`, which really could have just been `MatchersOrLiterals<Y>`